docs/how-to-guides/integrating-autoware/launch-autoware/perception.md

```diff
@@ -0,0 +1,33 @@
+# Perception mode
```

Comment:
Where did it come from?

Reply:
I found it from https://github.com/autowarefoundation/autoware_launch/blob/main/autoware_launch/launch/autoware.launch.xml#L39

Reply:
Ah ok you volunteered to add perception mode as well. Got it, thanks!

Reply:
I intentionally added that. But honestly, I don't have much knowledge about it, so the page doesn't have much content.